extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -386,3 +386,15 @@ export async function getFileShares(account: azdata.Account, subscription: azure
 		errors: response.errors ? response.errors : []
 	};
 }
+
+export async function getMigrationControllers(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, resourceGroupName: string, regionName: string, ignoreErrors: boolean): Promise<GetMigrationControllersResult> {
+	const apiEndpoint = `https://` + regionName + `.management.azure.com` +
+		`/subscriptions/${subscription.id}` +
+		`/resourceGroups/${resourceGroupName}` +
+		`/providers/Microsoft.DataMigration/Controllers/default/shares?api-version=2020-09-01-preview`;
```

Comment:
Where is this API version coming from? Is this something special just for migration or something?